source/language/classical.rst

```diff
@@ -110,6 +110,43 @@ multiplication, and division and the corresponding assignment operators.
    angle[10] c;
    c = angle(a + b); // cast to angle[10]
 
+
+Evaluation order
+~~~~~~~~~~~~~~~~
+
+OpenQASM evaluates expressions from left to right.
+
+   .. table:: [operator-precedence] operator precedence in OpenQASM.
+
+      +-------------------------------+---------------------------------------+
+      | Operator                      | Descriptions                          |
+      +-------------------------------+---------------------------------------+
+      | (), [], ++, --                | postfix for call, index, incrementors |
```

Comment:
```suggestion
      | (), ++, cast, literals, etc...                | Expression terminator |
```

Reply:
(I do know by the way that further operator precedence may need to be defined at the terminator level, but lets leave that for a future PR...)

Reply:
We do not need to list all the "expression terminator" in this table because some (such as literal and numbers) do not need operator precedence.

Reply:
Ok sure